tests/widget/inline/widget-device-profile.test.js

```diff
@@ -0,0 +1,16 @@
+import { expect, test } from '@playwright/test';
+
+import { asyncEvents } from '../../utilities/async-events.js';
+
+test('Inline widget with login in Canadian French', async ({ page, context }) => {
+  const { clickButton, navigate } = asyncEvents(page);
+  await navigate('widget/inline?journey=DeviceProfileNode');
+  await page.getByRole('textbox', { name: 'Username' }).type('demouser');
+  await page.getByRole('textbox', { name: 'Password' }).type('j56eKtae*1');
+  await clickButton('Next', '/authenticate');
+  await page.waitForRequest(/\/authenticate/, async () => {
+    const text = page.getByText('Collecting Profile');
+    await expect(text).toBeVisible();
+  });
+  await context.grantPermissions(['geolocation']);
```

Comment:
I feel like this test isn't finished. We don't seem to be asserting anything, just granting a permission. Can we write this so we assert something regarding device profile to ensure the profile is being generated?